index.markdown

```diff
@@ -6,16 +6,16 @@ title: Overview
 layout: home
 ---
 
-LODA is an assembly language, a computational model, and a distributed tool for mining programs. You can use it to generate and search programs that compute integer sequences from the [On-Line Encyclopedia of Integer Sequences®](https://oeis.org/) (OEIS®). The goal of the project is to find new formulas and more efficient algorithms for a wide range of non-trivial integer sequences.
+LODA is an assembly language, a computational model, and a distributed tool for mining programs. You can use it to generate and search programs that compute integer sequences from the [On-Line Encyclopedia of Integer Sequences®](https://oeis.org/) (OEIS®). The goal of the project is to reverse engineer formulas and efficient algorithms for a wide range of non-trivial integer sequences.
 
 ## The LODA Language
 
-LODA means _Lexicographical Order Descent Assembly_. It is an assembly-based language for solving number-theoretic problems. Its simple syntax combined with its powerful operations makes it ideal for automated search of new programs and algorithms using a process called _program mining_. In a nutshell, we use distributed compute power to generate random programs and match them against the OEIS database.
+LODA means _Lexicographical Order Descent Assembly_. It is an assembly-based language for solving number-theoretic problems. It has a simple syntax entirely focusing on arithmetic operations. This enables an automated search of new programs and algorithms using a process called _program mining_. In a nutshell, we use distributed compute power, smart search algorithms and machine-learning to find
```

Comment:
Replace `It has a simple syntax` with `It's has a mining-friendly syntax with arithmetic operations.`